package.json

```diff
@@ -76,7 +79,16 @@
     ],
     "rules": {
       "prettier/prettier": 2,
-      "no-shadow": 2
```

Comment:
this one is incompatible with typescript, the equivalent is already included:
see: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-shadow.md